includes/Settings.php

```diff
@@ -349,6 +359,28 @@ public function register(): void {
 				'show_in_rest' => true,
 			]
 		);
+
+		register_setting(
```

Comment:
Should really update unit tests [here](https://github.com/GoogleForCreators/web-stories-wp/blob/d87cf2a2f1a613c5a1a0e2f1d6ce8746b305ec87/tests/phpunit/integration/tests/Settings.php#L44).